sdk/textanalytics/ai-text-analytics/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 1.0.1 (Unreleased)
 
+- The `text` property of the `SentenceSentiment` interface is no longer optional because the service always returns it. This interface is used exclusively as an output type so this change does not break existing code.
```

Comment:
A slight update to communicate that nothing changed on the service side or in the actual data that is returned

```suggestion
- The `text` property of the `SentenceSentiment` interface is no longer marked as optional because the service has always returned it. This interface is used exclusively as an output type so this change does not break existing code.
```